module/zcommon/zfeature_common.c

```diff
@@ -348,6 +348,11 @@ zpool_feature_init(void)
 	    ZFEATURE_FLAG_MOS | ZFEATURE_FLAG_ACTIVATE_ON_ENABLE,
 	    ZFEATURE_TYPE_BOOLEAN, NULL);
 
+	zfeature_register(SPA_FEATURE_LIVELIST,
+	    "com.delphix:livelist", "livelist",
+	    "Improved clone deletion performance.",
```

Comment:
I think that this should depend on SPA_FEATURE_EXTENSIBLE_DATASET, since it adds DD_FIELD_LIVELIST.